sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/AzureCommunicationChatServiceClient.java

```diff
@@ -61,90 +39,66 @@
+    public PagedDataResponseCollection<ChatMessageReadReceipt, Page<ChatMessageReadReceipt>> listChatReadReceiptsWithPageResponse(String chatThreadId, Integer maxPageSize, Integer skip) {
+        return this.serviceClient.listChatReadReceiptsWithPageResponse(chatThreadId, maxPageSize, skip);
     }
 
     /**
-     * Gets read receipts for a thread.
-     * 
-     * @param chatThreadId Thread id to get the read receipts for.
+     * Gets chat message read receipts for a thread.
+     *
+     * @param chatThreadId Thread id to get the chat message read receipts for.
+     * @param maxPageSize The maximum number of chat message read receipts to be returned per page.
+     * @param skip Skips chat message read receipts up to a specified position in response.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws ErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return read receipts for a thread.
+     * @return chat message read receipts for a thread.
      */
-    public PagedDataResponseCollection<ReadReceipt, Page<ReadReceipt>> listChatReadReceiptsWithPageResponse(String chatThreadId) {
-        return this.serviceClient.listChatReadReceiptsWithPageResponse(chatThreadId);
-    }
-
-    /**
-     * Gets read receipts for a thread.
-     * 
-     * @param chatThreadId Thread id to get the read receipts for.
-     * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws ErrorException thrown if the request is rejected by server.
-     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return read receipts for a thread.
-     */
-    public PagedDataCollection<ReadReceipt, Page<ReadReceipt>> listChatReadReceiptsWithPage(String chatThreadId) {
-        return this.serviceClient.listChatReadReceiptsWithPage(chatThreadId);
+    public PagedDataCollection<ChatMessageReadReceipt, Page<ChatMessageReadReceipt>> listChatReadReceiptsWithPage(String chatThreadId, Integer maxPageSize, Integer skip) {
+        return this.serviceClient.listChatReadReceiptsWithPage(chatThreadId, maxPageSize, skip);
     }
 
     /**
      * Sends a read receipt event to a thread, on behalf of a user.
-     * 
+     *
      * @param chatThreadId Thread id to send the read receipt event to.
-     * @param body Request payload for sending a read receipt.
+     * @param sendReadReceiptRequest Request payload for sending a read receipt.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws ErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the response.
+     * @return the completion.
```

Comment:
I don't think this is very descriptive. Maybe we could use something like "An empty response indicating the operation's completion."